VAMobile/documentation/design/Components/Alerts and Progress/Alert.md

```diff
@@ -0,0 +1,75 @@
+
+On the website and mobile app, do not use the Alert component for:
+* **Long forms**. On long forms, always include in-line validation in addition to any error messages that appear at the top of the form.
+* **Destructive actions**. If an action will result in destroying a user’s work (for example, deleting an application) use a more intrusive pattern, such as a confirmation [modal](https://design.va.gov/components/modal) dialogue on the website or a native alert on the mobile app, to allow the user to confirm that this action is what they want.
+* **Clarifying background information**. Use the [Additional info](https://design.va.gov/components/additional-info) component when clarifying outcomes for an input or a form question as well as providing background information. Keep in mind that Alert - Expandable should warrant an alert and be used sparingly. The value of any type of alert is diminished if the page is littered with alerts of equal weight.
+
+#### Choosing between variations
+* Use standard alerts for most use cases.
+* Use expandable alerts when the information is not a response to user feedback.
+* Use dismissible alerts when the content is informational and not specific to the user or their interaction. For example, displaying “what’s new” content in the app.
+
+#### Placement
+* Alerts always appear near the top of the screen
+
+#### Content considerations
+* The [VA Design System's content considerations for alerts](https://design.va.gov/components/alert#content-considerations) are appropriate for the mobile app with the following addition:
+    * VA no longer says, “Please” in alerts when making a request of the user.
+* The VA Design System also includes guidance on how to write error alerts. In particular, the section on [style and tone](https://design.va.gov/patterns/help-users-to/recover-from-errors#style-and-tone) provides help on how to write clear and conversational alerts.
+
+##### Accessibility considerations
+* Alerts should only be used when appropriate to do so. Do not use Alerts when a native alert would be best (i.e. native alert, dialogue, action sheet, snackbar, etc.).
+* When using a screen reader, the Alert should announce itself as an alert with an indication of its role / importance (i.e. error, warning, informational, etc.).
+* Alerts that have expanded / closed states must be announced by a screen reader.
+* Do not automatically dismiss an Alert based on a timer or time limit.
+* Use alternative text (alt text) for any icons that are not considered decorative.
+* No buttons should be disabled within an Alert.
+* Focusable elements within an Alert should include: heading, body copy, phone numbers, and buttons.
+* [accessibilityLabel](https://reactnative.dev/docs/accessibility#accessibilitylabel) and [accessibilityLabelledBy](https://reactnative.dev/docs/accessibility#accessibilitylabelledby-android) should be used where appropriate.
+
+### Code usage
+Link to Storybook coming soon
```

Comment:
Just want to make sure we have an ticket open for this part somewhere? Or AC on another ticket

Reply:
Good call! Just created #7088 to add the Storybook links later.